brave/brave-core
# Use brave-browser-stable as the /usr/bin/ symlink for release packages

This fixes brave/brave-browser#1324 by ensuring that the release Debian package can also provide an alternative (of higher priority) for the /usr/bin/brave-browser binary.

With this in place, that binary will default to the release browser and it can be configured normally using:

    sudo update-alternatives --config brave-browser

As [noted previously](https://github.com/brave/brave-browser/issues/1324#issuecomment-476423994), the RPMs don't need to be fixed since they already exhibit the correct behavior.

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests && npm run test-security`) on

## Test Plan:

I tested the following "upgrade" scenario on Ubuntu 18.04:

1. Install current stable release (0.61) and then current beta (0.62).
2. Confirm that the /usr/bin/brave-browser binary is a symlink to
   /etc/alternatives/brave-browser which in turn is a symlink to beta.
3. Confirm that `update-alternatives --config brave-browser` only offers beta.
4. Upgrade the stable package to the one in this PR.
5. Confirm that the /usr/bin/brave-browser-stable binary now exists.
6. Confirm that the /etc/alternatives/brave-browser symlink now points to stable and that `update-alternatives --config brave-browser` offers both stable and beta.
7. `apt purge brave-browser` and confirm that
   /etc/alternatives/brave-browser symlink now points to beta and that
   `update-alternatives --config brave-browser` only offers beta.

and this simpler "new install" scenario:

1. Purge all existing brave-browser* packages.
2. Install the stable package from this PR.
3. Confirm that the /etc/alternatives/brave-browser symlink points to stable and that `update-alternatives --config brave-browser` offers only stable.
4. Confirm that typing `brave-browser` launches the browser.

Finally, I built a beta version of this package and ran through this scenario:

1. Install the stable package from this PR.
2. Install the current beta (0.62) package from the official repo.
3. Confirm that the /etc/alternatives/brave-browser symlink points to stable and that `update-alternatives --config brave-browser` offers stable and beta.
4. Upgrade brave-browser-beta with the package from this PR.
5. Confirm that the /etc/alternatives/brave-browser symlink still points to stable and that `update-alternatives --config brave-browser` also offers stable and beta.

## Reviewer Checklist:
